public/js/components/Frames.js

```diff
@@ -10,7 +10,9 @@ const { endTruncateStr } = require("../util/utils");
 const { basename } = require("../util/path");
 const { getFrames, getSelectedFrame, getSource } = require("../selectors");
 
-require("./Frames.css");
+if (typeof window == "object") {
+  require("./Frames.css");
+}
```

Comment:
Yeah, might be nice to do it in this PR if it's a quick fix. If it's not for some reason, you can look into it later.